### DISCUSSION TITLE
`useFieldArray` does not update `errors` object in form state on its field change.

### DISCUSSION THREAD

**Discussion by DarryQueen** [react-hook-form/react-hook-form#10038] [Q&A]
### Version Number

7.43.3

### Codesandbox/Expo snack

https://codesandbox.io/s/pedantic-wing-f0sv73?file=/src/App.tsx

### Steps to reproduce

1. Add a few numbers whose total is _below_ 10. You should see an error at the top saying the total needs to be at least 10.
2. Edit the numbers such that the total is at least 10. The error persists.
3. Click "submit" to trigger validation. The error disappears.

### Expected behaviour

I expect the errors object to be updated, given that the validation is called in the background (note that the form state's `isValid` is live-updated).

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
`useFieldArray` rules are at the field array level instead of the field level, which validation is performed at each user action.  However, you can use `trigger` instead.
